README.md

```diff
@@ -0,0 +1,140 @@
+# Babylon React Native
```

Comment:
We probably need to follow Microsoft open source guidelines here as the plan is to do this similar to Babylon Native repo.

Reply:
I'm not sure what that means. Is there something you want me to change now?

Reply:
No, not yet.